src/libraries/System.Reflection.MetadataLoadContext/src/System/Reflection/TypeLoading/General/TypeExtensions.netcoreapp.v7.cs

```diff
@@ -0,0 +1,70 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Is netcoreapp.v7 the common pattern with file names? I assumed that netcoreapp was a deprecated term. Perhaps this should just be "net7"

Reply:
There were already "TypeExtensions.netcoreapp.cs" and "TypeExtensions.netstandard" so I followed those. But renaming to "net" and "net7" makes sense.